src/Traits/VerifiableTrait.php

```diff
@@ -15,20 +14,33 @@ trait VerifiableTrait
 
     protected function loadActiveVerifications(string $action)
     {
-        $newVerifications = $this->shouldKeepDuringSession($action)
-            ? VerificationCode::allFor($this)->whereNull('verifies_until')->whereNotNull('used_at')->get()
-            : VerificationCode::allFor($this)->where('verifies_until', '>', Carbon::now()->toDateTime())->get();
+        $newVerifications = $this->getNewVerifications($action);
```

Comment:
tejto metodka nerozumiem, ze co si chcel docielit, ale zda sa mi to byt trosku somarina

Reply:
Tato metoda loaduje aktivne verifikacie. Na zaklade expires_from configu pre danu akciu vybera z DB a predovsetkym pri **expires_from == 'last-activity'** updatuje atributy **last_touched** a **verifies_until**. Nasledne ich merguje s kolekciou povodnych verifikacii, ak tam nejake su.

Reply:
O tomto si podme zavolat v pondelok, nerozumiem presne ako toto fugnuje a na co to potrebujeme